src/admin-views/widgets/components/checkbox.php

```diff
@@ -0,0 +1,39 @@
+<?php
+/**
+ * Admin View: Widget Checkbox Component
+ *
+ * Override this template in your own theme by creating a file at:
+ * [your-theme]/tribe/admin-views/widgets/components/checkbox.php
+ *
+ * See more documentation about our views templating system.
+ *
+ * @link http://m.tri.be/1aiy
+ *
+ * @var string  $label   Label for the checkbox.
+ * @var string  $value   Value for the checkbox.
+ * @var string  $id      ID of the checkbox.
+ * @var string  $name    Name attribute for the checkbox.
+ * @var boolean $checked Whether the checkbox is checked or not.
+ *
+ * @version TBD
+ *
+ */
+?>
```

Comment:
* :no_entry_sign: There must be exactly one blank line after the file comment
* :no_entry_sign: Additional blank lines found at end of doc comment

<sub>hash: 2d8179f908455d14be8437d64714037a</sub>